Documentation/kubernetes/troubleshooting.rst

```diff
@@ -120,3 +120,12 @@ Now if you try to create a CNP as a TPR, you will get an error:
 .. code:: bash
 
        $ Error from server (BadRequest): error when creating "cilium-tpr.yaml": the API version in the data (cilium.io/v1) does not match the expected API version (cilium.io/v2)
+
+Apiserver outside of cluster
+==============================
+
+If you are running Kubernetes Apiserver outside of your cluster for some reason (like keeping master node behind firewall), make sure that you run Cilium on master node too.
+Otherwise kubernetes pod proxies created by apiserver will not be able to route to cluster IPs and you may encounter errors when trying to proxy traffic to pods.
```

Comment:
are cluster IPs here the same thing as ClusterIPs? If so change the spacing / casing to match the nomenclature set forth by Kubernetes